packages/tabs/src/useTabs.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { useState } from 'react';
+import { useSelection } from '@zendeskgarden/container-selection';
+import { generateId } from '@zendeskgarden/container-utilities';
+
+function requiredArguments(arg, argStr, methodName) {
+  if (typeof arg === 'undefined' || arg === null) {
+    throw new Error(
+      `Accessibility Error: You must provide an "${argStr}" option to "${methodName}()"`
+    );
+  }
+}
+
+export function useTabs({ vertical, idPrefix, ...options } = {}) {
+  const { selectedItem, focusedItem, getContainerProps, getItemProps } = useSelection({
+    direction: vertical ? 'vertical' : 'horizontal',
+    ...options
+  });
+  const [_id] = useState(idPrefix || generateId('garden-tabs-container'));
+  const panelId = `${_id}--panel`;
+  const tabId = `${_id}--tab`;
+
+  const getTabListProps = ({ role = 'tablist', ...other } = {}) => {
```

Comment:
Should this receive an `aria-orientation`? https://www.w3.org/TR/wai-aria-practices/#tabpanel

Reply:
oh good catch I wonder if I should add this to the underlying container-selection instead?

Reply:
See #35

Reply:
bumping container-selection now spreads `aria-orientation` see d8f75a6